system_metrics_collector/src/system_metrics_collector/collector.cpp

```diff
@@ -24,58 +24,58 @@
 namespace system_metrics_collector
 {
 
-bool Collector::start()
+bool Collector::Start()
 {
-  std::unique_lock<std::mutex> ulock(mutex);
+  std::unique_lock<std::mutex> ulock(mutex_);
   if (started_) {
     return false;
   }
   started_ = true;
-  return setupStart();
+  return SetupStart();
 }
 
-bool Collector::stop()
+bool Collector::Stop()
 {
   bool ret = false;
   {
-    std::unique_lock<std::mutex> ulock(mutex);
+    std::unique_lock<std::mutex> ulock(mutex_);
```

Comment:
```c++
std::unique_lock<std::mutex> ulock{mutex_};
```

Reply:
Yes, it does. Switched over to using curly brace initialization.